src/stores/settings.js

```diff
@@ -55,7 +55,9 @@ export const useSettingsStore = defineStore('settings', {
 
 	getters: {
 		getShowMediaSettings: (state) => (token) => {
-			if (state.showMediaSettings[token] !== undefined) {
+			if (!token) {
+				return true
+			} else if (state.showMediaSettings[token] !== undefined) {
```

Comment:
Personally I think that `if` clauses with returns are clearer on their own, like:
```
			if (!token) {
				return true
			}

			if (state.showMediaSettings[token] !== undefined) {
```

But this is just style nitpicking, so also fine by me to keep the current code.